apps/theming-designer/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html dir="ltr">
```

Comment:
Nit, you may also want to specify `lang="en"` here unless its handled dynamically by our library.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang